### DISCUSSION TITLE
SCI model

### DISCUSSION THREAD

**Discussion by mdejolier** [Green-Software-Foundation/carbon-ci#41] [General]
I am very interested in SCI, I developed a green IT data center model 12 years ago and I am currently adding SCI support to it. However, I think that capacity planning is missing here. The power consumption of a server is not linear, e.g. if it uses 100 watts/hour when Idle, it uses 700 watts/hour at 100 % use, but the inflexion point is around 50 % usage. So, it would seem that there is a big Co2 avoidance if you use the server up to its limits, but the reality is that the performance degrades quickly above 80 %. The usage of a simple M/M/1 model gives a queue time of 4 times the active time, although I admit that M/M/1 is pessimistic and M/G/1 or M/G/K/T could give a more accurate result. I have been a capacity planning expert in MVS modeling and my software tool was acquired at that time by Legent Corp (Mario Morino) and sold worldwide to large companies using IBM MVS mainframes.

**Comment by NAMRATA-WOKE** [comment]
@Green-Software-Foundation/sci-contributors maybe one of you can chime in?
